### ISSUE TITLE
Black texture when model display on screen

### ISSUE THREAD

**Issue by claireliu14** [ThomasGorisse/sceneform-android-sdk#46]
There will be a black shadow when the model appears. As shown in the [video](https://drive.google.com/file/d/19J1gcbYfsskONzpDNQYgNoor9ZgoZpze/view?usp=sharing)
The andy model is downloaded from [Poly](https://poly.google.com/view/9-bJ2cXrk8S).
The code is as below :
```java
    private void loadModel() {
        WeakReference<MainActivity> weakActivity = new WeakReference<>(this);
        ModelRenderable.builder()
                .setSource(this, Uri.parse("andy.glb"))
                .setIsFilamentGltf(true)
                .build()
                .thenAccept(modelRenderable -> {
                    MainActivity activity = weakActivity.get();
                    if (activity != null) {
                        activity.modelRenderable = modelRenderable;
                    }
                })
                .exceptionally(throwable -> {
                    Log.e(TAG, throwable.getMessage());
                    return null;
                });
    }

    @Override
    public void onTapPlane(HitResult hitResult, Plane plane, MotionEvent motionEvent) {
        if (modelRenderable == null) {
            return;
        }
        arFragment.getArSceneView().getScene().addOnUpdateListener(this);

        Anchor anchor = hitResult.createAnchor();
        AnchorNode anchorNode = new AnchorNode(anchor);
        anchorNode.setParent(arFragment.getArSceneView().getScene());

        Node node = new Node();
        node.setParent(anchorNode);
        node.setRenderable(modelRenderable);
        node.setLocalScale(Vector3.one().scaled(0.1f));

        FilamentAsset assets = node.getRenderableInstance().getFilamentAsset();
        if (assets.getAnimator().getAnimationCount() > 0) {
            Log.e(TAG, "animators count: ".concat(String.valueOf(assets.getAnimator().getAnimationCount())));
            animators.add(new AnimationInstance(assets.getAnimator(), 0, System.nanoTime()));
        }
    }

    @Override
    public void onUpdate(FrameTime frameTime) {
        Long time = System.nanoTime();
        for (AnimationInstance animatorInstance : animators) {
            animatorInstance.animator.applyAnimation(
                    animatorInstance.index,
                    ((float) (time - animatorInstance.startTime) / TimeUnit.SECONDS.toNanos(1)) % animatorInstance.duration
            );
            animatorInstance.animator.updateBoneMatrices();
        }
    }
```

My question is that sceneform problem or model problem ?
Anyone met this problem before ?

**Comment by ray650128** [comment]
I have the same problem, how can I fix it?

**Comment by ThomasGorisse** [comment]
I think it has been fixed it recently.
Can you try using the snapshot version to test before I publish a new release :
*/build.gradle*
```gradle
allprojects {
    repositories {
        ...
        maven { url 'https://jitpack.io' }
    }
}
```

*app/build.gradle*
```gradle
dependencies {
    implementation 'com.github.ThomasGorisse.sceneform-android-sdk:sceneform:-SNAPSHOT'
}
```

**Comment by claireliu14** [comment]
@ThomasGorisse 
It tried the repo, but still had the same problem.
The screen recording is [here](https://drive.google.com/file/d/19klvnZHvyVzoqyDOxKunicc2yEeM8t8L/view?usp=sharing).

[andy glb model](https://github.com/ThomasGorisse/sceneform-android-sdk/files/6381491/andy.zip)

**Comment by ThomasGorisse** [comment]
Thanks for the report.

That's actually a normal behavior since, for performance reasons, textures are load asynchronously (after the base model loading).
I'll add a parameter to the `ModelRenderable.Builder` to disable the async loading feature and make the `thenAccept` callback call after everything is load. The counterpart is that the loading time will increase.
I'll let you know when the 1.18.10 is available.

Thanks for your sharing.

**Comment by ThomasGorisse** [comment]
@claireliu14 and @ray650128 Can you test if everything is fixed in the 1.18.10 ?

**Comment by claireliu14** [comment]
@ThomasGorisse I tried `.setAsyncLoadEnabled(false)` int the 1.18.10 and it worked fine!
Thanks for your support.
